### ISSUE TITLE
Bump dependencies to recent versions

### ISSUE THREAD

**Issue by ju5t** [sensson/puppet-powerdns#63]
The module is a little behind when it comes to its dependencies. Unfortunately, while testing recent versions of the dependencies I ran into an issue with `puppetlabs/mysql` version 6 taking up all CPU and memory.

Waiting on feedback: [MODULES-8158](https://tickets.puppetlabs.com/browse/MODULES-8158).

**Comment by ju5t** [comment]
Closing in favour of #68
